crypto_kem/ntrulpr761/clean/kem.c

```diff
@@ -156,7 +158,9 @@ static void Hide(unsigned char *c, unsigned char *r_enc, const Inputs r, const u
             s[0] = 5;
             Hash(h, s, sizeof s);
         }
-        PQCLEAN_NTRULPR761_CLEAN_crypto_stream_aes256ctr((unsigned char *) L, 4 * p, aes_nonce, h);
+        if (PQCLEAN_NTRULPR761_CLEAN_crypto_stream_aes256ctr((unsigned char *) L, 4 * p, aes_nonce, h) != 0) {
+            abort();
+        }
```

Comment:
could this `abort()` not better be handled in the function that wraps `aes256ctr`?

Reply:
Thanks. Upstream might call OpenSSL's aes directly at that line, but the abort isn't necessary with the wrapper. I've removed it.